database/src/lib.rs

```diff
@@ -72,6 +72,159 @@
 // ).await?,
 
 use scylla::prepared_statement::PreparedStatement;
+use scylla::retry_policy::{QueryInfo, RetryDecision};
+use scylla::transport::errors::QueryError;
+
+#[derive(Debug)]
+pub struct CustomRetryPolicy {
```

Comment:
Since it is related to the database I'd suggest renaming it to `CustomDBRetryPolicy` to be more explicit